packages/runtime/src/shared/common.ts

```diff
@@ -0,0 +1,18 @@
+import type { EnsembleScreenModel } from "@ensembleui/react-framework";
+import {
+  evaluate,
+  replace,
+  visitExpressions,
+} from "@ensembleui/react-framework";
+
+export const evaluateInputs = (
```

Comment:
Put this in framework. Rename to `evaluateDeep` so it is more general and rename variables to reflect more general use cases.

Reply:
I already tried this by adding `evaluateInputs` into common.ts of framework, but it will create cycling dependancy, and only for that I created new common in runtime.

we are importing evaluate from evaluate.ts, which also import other methods from common.ts, which will create the cycling dependancy